paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -715,68 +748,106 @@ def status_code_by_state(self) -> Mapping[str, int]:
 
         return codes
 
+    def is_terminal_state(self, state: str) -> bool:
+        return (state in self.status_code_by_state())
+
+    def get_available_buttons(self) -> List[str]:
+        buttons = []
+
+        if self.is_terminal_state(self.state):
+            # If we're about to exit, always clear the buttons, since once we exit the buttons will stop working.
+            return []
+
+        if self.old_git_sha != self.commit:
+            if self.state in self.rollback_states:
```

Comment:
I was kinda hoping we could have both buttons consistently on or off, even if the other button doesn't "do" anything, for the sake of UI stability and not making buttons disappear out from underneath you. 

What do you think? Would it be less confusing to have a button that doesn't do anything, or would it be less confusing to only have a button that can do something?

Reply:
I think it would be confusing to have buttons that do nothing. Especially since they wouldn't cause us to re-mark-for-deployment or anything that might help us recover from weird situations like "someone ran `paasta rollback` in a different terminal".